modules/util/pathfinder/module.yaml

```diff
@@ -0,0 +1,34 @@
+schema_version: 1
+
+name: "jboss.container.util.pathfinder"
+version: "1.0"
+description: ^
+  "Provides support for determining Java application jar locations
+   as well as library directories"
+
+envs: 
+- name: JBOSS_CONTAINER_UTIL_PATHFINDER_MODULE
+  value: /opt/jboss/container/util/pathfinder
+
+- name: JAVA_APP_DIR
```

Comment:
please also remove the documentation for these three environment variables from the old location in `modules/run/module.yaml`.